jetty-ee8/jetty-ee8-websocket/jetty-ee8-websocket-jetty-client-webapp/src/main/config/modules/ee8-websocket-jetty-client.mod

```diff
@@ -0,0 +1,27 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Expose the Jetty WebSocket Client classes to deployed web applications.
+
+[environment]
+ee8
+
+[tags]
+websocket
+
+[depend]
+client
+ee8-annotations
+
+[lib]
+lib/jetty-websocket-core-common-${jetty.version}.jar
+lib/jetty-websocket-core-client-${jetty.version}.jar
+lib/ee8-websocket/jetty-ee8-websocket-jetty-api-${jetty.version}.jar
+lib/ee8-websocket/jetty-ee8-websocket-jetty-common-${jetty.version}.jar
+lib/ee8-websocket/jetty-ee8-websocket-jetty-client-${jetty.version}.jar
+lib/ee8-websocket/jetty-ee8-websocket-jetty-client-webapp-${jetty.version}.jar
```

Comment:
I was expecting to see a jetty-ee8-websocket-jetty-client-webapp.mod file which would declare the jetty-ee8-websocket-jetty-client-webapp.jar, and would depend on the jetty-ee8-websocket-jetty-client.mod. But I suppose it's ok like this.

Reply:
I don't see someone using `jetty-ee8-websocket-jetty-client.mod` without `jetty-ee8-websocket-jetty-client-webapp.mod` or the other way around. So I think they should be in the same mod file.

Reply:
We always follow the convention that the Jetty module name is derived from the Maven module name, so this should be renamed to `*-webapp.mod`.